config.toml

```diff
@@ -152,10 +152,10 @@ no = 'Sorry to hear that. Please <a href="https://github.com/navidrome/website/i
 	icon = "fab fa-discord"
         desc = "Chat with other users and project developers"
 [[params.links.user]]
-	name = "Twitter"
-	url = "https://twitter.com/navidrome"
-	icon = "fab fa-twitter"
-        desc = "Follow us on Twitter"
+	name = "X"
+	url = "https://x.com/navidrome"
+	icon = "fab fa-x"
```

Comment:
The X logo is actually fa-x-twitter (see: https://fontawesome.com/icons/x-twitter?f=brands&s=solid)

Reply:
made the change @deluan 
merge if passes the check.